src/Umbraco.Web.UI.Client/src/common/directives/components/tree/umbtree.directive.js

```diff
@@ -407,6 +407,168 @@ function umbTreeDirective($compile, $log, $q, $rootScope, treeService, notificat
                         lastSection = newVal;
                     }
                 });
+				
+                scope.sortableOptions = {
+                    connectWith: ".item",
+                    cursor: "move",
+                    items: '>li',
+                    axis: 'y',
+                    tolerance: 'pointer',
+                    containment: '.umb-tree .root>ul',
+                    delay: 300,
+                    disabled: !scope.section.match("content|media") || scope.isdialog === "true",
+                    update: function (e, ui) {
+                        if (!scope.section.match("content|media") || scope.isdialog === "true")
+                            return;
+
+                        var node = ui.item.scope() ? ui.item.scope().node : scope.dragCurrentNode;
+
+                        scope.newParentNode = $(e.target.parentElement).scope().node;

```

Comment:
Any reason why many of these variables: newParentNode, dragCurrentNode, dragMoved, originalSort are on the scope? From the looks of things this should be a non scoped normal js object variable to store the state instead of adding additional properties to the scope.